# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../cloop/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-06-24 18:09:20.000000000 -0400
+++ ./Makefile	2007-06-24 18:09:53.000000000 -0400
@@ -3,7 +3,7 @@
 KVERSION=$(shell uname -r)
 KERNEL_DIR=/lib/modules/$(KVERSION)/build
 
-DESTDIR=/usr/local
+DESTDIR=
 #INSTALL_MOD_PATH:=/tmp
 
 # Add compressloop when it has been fixed
